ducktape/command_line/main.py

```diff
@@ -36,6 +36,7 @@ def parse_args():
     parser.add_argument("--collect-only", action="store_true", help="display collected tests, but do not run")
     parser.add_argument("--debug", action="store_true", help="pipe more verbose test output to stdout")
     parser.add_argument("--exit-first", action="store_true", help="exit after first failure")
+    parser.add_argument("--no-teardown", action="store_true", help="do not stop and clean services when test finishes")
```

Comment:
The help string could also explain why and when you would do this. It's really meant for test developers, not during normal test runs.